calib:
  cam1_cam2:
    matching:
      iss_nms_radius: 15.0
      iss_support_radius: 15.0
      kp_dupl_rej: true
      kp_est_radius: 20.0
      match_th: 1.0
      match_th_dist: 100.0
      normal_radius: 50.0
      x_enable: true
      x_min: -265.0
      x_max: 5000.0
      y_enable: false
      y_min: -505.0
      y_max: 395.0
      z_enable: true
      z_min: 30.0
      z_max: 800.0